updates/ryan.sql

```diff
@@ -0,0 +1,21 @@
+DELETE FROM selektion_ort WHERE ID BETWEEN 39 AND 60;
```

Comment:
Note: I already executed this on the servers, so i will rename the PR since this is no longer its main purpose